astropy/utils/misc.py

```diff
@@ -224,12 +224,17 @@ def find_api_page(obj, version=None, openinbrowser=True, timeout=None):
     elif version == 'dev' or version == 'latest':
         baseurl = 'http://devdocs.astropy.org/'
     else:
-        baseurl = f'http://docs.astropy.org/en/{version}/'
+        baseurl = f'https://docs.astropy.org/en/{version}/'
+
+    # Custom request headers; see
+    # https://github.com/astropy/astropy/issues/8990
+    req = urllib.request.Request(
+        baseurl + 'objects.inv', headers={'User-Agent': 'Astropy/{version}'})
```

Comment:
Added the custom headers that @saimn suggested. Thanks, Simon!

Reply:
@pllim - Should `'Astropy/{version}'` be an f-string `f'Astropy/{version}'` ?

Reply:
Ops! You are right! :sweat_smile:  You can see that I was not one of the early adopters...